src/main/grammars/ElmParser.bnf

```diff
@@ -314,11 +311,9 @@ private Atom ::=
 // Various tokens that can begin a new expression. Useful for parse error recovery.
 private expr_delimiters ::= COMMA | LEFT_BRACE | LEFT_PARENTHESIS | LEFT_SQUARE_BRACKET
 
-ExpressionWithAccessorExpr ::=
-    ParenthesizedExpr ExpressionAccessor
-
-RecordWithAccessorExpr ::=
-    RecordExpr ExpressionAccessor
+FieldAccessExpr ::= FieldAccessStart FieldAccessSegment+
+FieldAccessStart ::= LOWER_CASE_IDENTIFIER | ParenthesizedExpr | RecordExpr
```

Comment:
I don't know if this is viable, but the Psi would probably be easier to work with if, rather than creating this wrapper rule, we did something like this:
```
private FieldAccessStart ::= ValueExpr | ParenthesizedExpr | RecordExpr
```